### PR TITLE
[4.0] Versions toolbar button

### PR DESCRIPTION
**Pull request by brianteeman**
Update the layout for the versions toolbar button to use the custom element used by the other toolbar buttons

Testing by code review

Pull Request for part of Issue #24850 .

### CODE REVIEW

File: layouts/joomla/toolbar/versions.php [hunk 1]
```diff
@@ -48,9 +48,13 @@
 	)
 );
 ?>
-<button<?php echo $id ?? ''; ?>
-	onclick="document.getElementById('versionsModal').open()"
-	class="btn btn-primary"
-	data-toggle="modal">
-	<span class="fa fa-code-branch" aria-hidden="true"></span><?php echo $title; ?>
-</button>
+<joomla-toolbar-button id="toolbar-versions">
```
Comment (Fedik) [inline layouts/joomla/toolbar/versions.php:54]: I think it need to be with `<?php echo $id ?? ''; ?>`, as before

Comment (brianteeman) [reply layouts/joomla/toolbar/versions.php:54]: I thought about that but isnt that just when you have a generic button that you need the variable. This one is a unique button

Comment (brianteeman) [reply layouts/joomla/toolbar/versions.php:54]: Also that code doesn't produce an id at all :(

Comment (Fedik) [reply layouts/joomla/toolbar/versions.php:54]: maybe it fine then, 
maybe previous version was a copy/paste from somewhere ;)

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3973061c796736ea5d3566ac7b8fc906b796f89b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24852">issues.joomla.org/tracker/joomla-cms/24852</a>.</sub>

File: layouts/joomla/toolbar/versions.php (hunk 1 above)
Comment (brianteeman) [reply layouts/joomla/toolbar/versions.php:54]: yes it looks like it

**Comment by wilsonge** [comment]
We need the `HtmlHelper` imports please

```
HTMLHelper::_('behavior.core');
HTMLHelper::_('webcomponent', 'system/joomla-toolbar-button.min.js', ['version' => 'auto', 'relative' => true]);
```

**Comment by brianteeman** [comment]
yeah - sorry - I spotted that last night and forgot to update

**Comment by wilsonge** [comment]
Thanks!

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**